src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/entity_search_input.html.twig

```diff
@@ -29,10 +29,11 @@
     'data-prototype-template': form_row(prototype),
     'data-prototype-index': prototype.vars.name|e('html_attr'),
     'data-prototype-mapping': prototype_mapping|json_encode,
-    'data-remove-modal': remove_modal|json_encode,
+    'data-remove-modal': remove_modal ? remove_modal|json_encode : null,
     'data-remote-url': remote_url,
     'data-data-limit': limit,
-    'data-allow-delete': form.vars.allow_delete ? 1 : 0
+    'data-allow-delete': form.vars.allow_delete ? 1 : 0,
```

Comment:
Im not sure its relevant in this PR.
I still need to be able to delete the selected element, I just don't need to show the confirmation modal.

Reply:
Confirmation modal doesn't really make sense here, as its just a selection in a form (we just use search input because there are lots of customers so we cannot rely on simple choice field). So you just want to select another option (another customer). And changes takes effect only after hitting save.

Reply:
btw I think its nice to have the confirmation modal optional in this search component. As before it was forced everywhere.

Reply:
It was initially designed so that the modal was always present Because I understood that every deletion action must be associated with a confirmation modal in every part of the BO, so it's one purpose (even if an additional submit is required)

Even in SEO tab when the selected category is removed the confirmation modal pops up, deletion is always confirmed however replacement happens automatically